test/livebook/intellisense_test.exs

```diff
@@ -410,6 +415,69 @@ defmodule Livebook.IntellisenseTest do
              ] = Intellisense.get_completion_items("mod.ve", binding, env)
     end
 
+    # TODO: Enable on Elixir 1.13
```

Comment:
Create a tag instead and exclude it on your test helper based on the version? :)

Reply:
On Elixir 1.13 some other tests fail because of changes in formatter or signatures, so this suite will need to be adjusted once the new version is out, and before that it's not really practical to run it. Thoughts?

Reply:
A TODO sounds good to me then!